frontend/src/styles/VendorCard.css

```diff
@@ -64,3 +64,58 @@
   display: -webkit-box;
   -webkit-line-clamp: 3;
 }
+
+@media screen and (max-width:300px){
+  .vendorDescription, .vendorName{
+    max-width:90px;
+  }
+
+  .vendorCard{
+      height: auto;
+  }
+}
+
+@media screen and (min-width:400px){
+  .vendorCard {
+    display: grid;
+    grid-template-columns: 1fr 2fr;
+    grid-template-rows: 1fr 2fr;
+  }
+  .vendorDescription, .vendorName{
+    max-width: 220px;
+    padding-left: 25px;
+  }
+}
+
+@media screen and (min-width:600px) and (max-width:809px){
+    .card-list{
+    justify-content:  center;
+    }
+    .vendorCard{
+      width: 36em;
+    }
+}
+
+@media screen and (min-width:430px) and (max-width:600px){
+  .card-list{
+  justify-content:  center;
+  }
+  .vendorCard{
+  width: 26em;
+  }
+}
+
+@media screen and (min-width:1000px) and (max-width:1200px){
```

Comment:
És miért optimalizálunk erre a felbontásra?

Reply:
ezekkel a méretekkel néznek ki szépen rekurzívan és lebontásra kerültek a különböző felbontásoknak megfelelően

Reply:
Mi volt a gond azzal az állapottal, ami most a stage-en látható, ahol mindig 100% szélesek a kártyák?

Reply:
Illetve ez melyik issue? Milyen bugot javít?